### PR TITLE
Add changelog of e5.5.1

### PR DESCRIPTION
**Pull request by yanzhiemq**

### CODE REVIEW

**Comment by JimMoen** [comment]
plz take a look about https://github.com/emqx/emqx/pull/12606

File: en_US/changes/changes-ce-v5.md [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 - [#12606](https://github.com/emqx/emqx/pull/12606) The Prometheus API experienced crashes when the specified SSL certificate file did not exist in the given path. Now, when an SSL certificate file is missing, the `emqx_cert_expiry_at` metric will report a value of 0, indicating the non-existence of the certificate.
 
+- [#12620](https://github.com/emqx/emqx/pull/12620) Fixed an issue when sensitive headers for HTTP connector may be printed in the `debug` level log.
```
Comment (Meggielqk) [inline en_US/changes/changes-ce-v5.md]: This one seems to describe a similar issue as in #12542. Are they duplicated?

Comment (id) [reply en_US/changes/changes-ce-v5.md]: @Meggielqk Good point. #12620 is a more general fix for the same issue. I suggest to leave just that one in the changelog and remove #12542.

File: en_US/changes/changes-ce-v5.md [hunk 2]
```diff
@@ -24,7 +22,7 @@
 
 - [#12606](https://github.com/emqx/emqx/pull/12606) The Prometheus API experienced crashes when the specified SSL certificate file did not exist in the given path. Now, when an SSL certificate file is missing, the `emqx_cert_expiry_at` metric will report a value of 0, indicating the non-existence of the certificate.
 
-- [#12620](https://github.com/emqx/emqx/pull/12620) Fixed an issue when sensitive headers for HTTP connector may be printed in the `debug` level log.
+- [#12620](https://github.com/emqx/emqx/pull/12620) Redacted authorization headers to exclude basic authorization credentials from `debug` level logs in the HTTP Server connector, mitigating potential security risks.
```
Comment (id) [inline en_US/changes/changes-ce-v5.md]: It's not just basic authorization, we redact all kinds of credentials.
```suggestion
- [#12620](https://github.com/emqx/emqx/pull/12620) Redacted sensitive information in HTTP headers to exclude authentication and authorization credentials from `debug` level logs in the HTTP Server connector, mitigating potential security risks.
```

File: en_US/changes/all-changes-ce.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ The release notes page for EMQX provides a comprehensive and detailed record of
 
 ## v5.5
 
+- [5.5.1](./changes-ce-v5.md#_5-5-1): 2024-03-05
```
Comment (wivwiv) [inline en_US/changes/all-changes-ce.md]: 2024/03/06?

**Review by wivwiv — APPROVED** [review]

**Pull request merged**